util/stderr_logger.cc

```diff
@@ -19,9 +19,10 @@ void StderrLogger::Logv(const char* format, va_list ap) {
   const time_t seconds = now_tv.tv_sec;
   struct tm t;
   port::LocalTimeR(&seconds, &t);
-  fprintf(stderr, "%04d/%02d/%02d-%02d:%02d:%02d.%06d %llx ", t.tm_year + 1900,
-          t.tm_mon + 1, t.tm_mday, t.tm_hour, t.tm_min, t.tm_sec,
-          static_cast<int>(now_tv.tv_usec),
+
+  fprintf(stderr, "%s%04d/%02d/%02d-%02d:%02d:%02d.%06d %llx ",
+          log_prefix.c_str(), t.tm_year + 1900, t.tm_mon + 1, t.tm_mday,
+          t.tm_hour, t.tm_min, t.tm_sec, static_cast<int>(now_tv.tv_usec),
```

Comment:
I don't feel strongly about it, but I prefer timestamp is the first thing of the line, so that they are aligned when shown on screen. log_prefix can be after timestamp.

Reply:
This is reasonable. If you have multiple DBs (which the prefix assumes, anyway), a common prefix is certainly nicer.